src/components/admin/tuits-manager/editable-tuit.js

```diff
@@ -47,35 +48,16 @@ const EditableTuit = ({tuit}) => {
                 }
             </div>
             <div className="w-100">
-                {
-                    !editing &&
-                    <i data-testid='edit-button'
-                       className='float-end fa fa-pen edit-button'
-                       onClick={() => {
-                           setEditing(true)
-                       }}/>
-                }
-                {
-                    editing &&
-                    <div className='up-del-buttons'>
-                        <i onClick={() => {
-                            setEditing(false)
-                        }}
-                           className="float-end fa fa-check save-button"/>
-                        <i
-                            onClick={() => {
-                            }}
-                            className="float-end fa fa-trash me-1 trash-button"/>
-                    </div>
-                }
+                <i onClick={()=> deleteTuit(tuit._id)}
+                className='float-end fa fa-trash mr-1'/>
```

Comment:
Would be better for there to be a button (similar to the pencil button before) to press before you can see the trash can so the admin would not accidentally delete a tuit.

Reply:
We used to have a pencil icon because it include both update and delete icons. As we discussed in the chat, admin shouldn't update user's tuit. The only thing admin can do is to delete tuits. Therefore, I don't think the pencil is necessary. We can have a window for admin to confirm deleting tuits, but it's another feature that should be implemented in another branch. Let me know what you think!

Reply:
I was thinking like a three dot or gear icon instead of a screen, just so they have to click twice if they really wanted to delete something

Reply:
I feel it's redundant. I didn't see apps designed like this. @caijiah thoughts?